01week/rockPaperScissors.js

```diff
@@ -8,34 +8,35 @@ const rl = readline.createInterface({
 });
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase().trim();
+  hand2 = hand2.toLowerCase().trim();
   if(hand1 === 'rock' && hand2 === 'paper'){
-    return 'hand2 wins';
+    return "Hand two wins!";
   }
   else if(hand1 === 'rock' && hand2 === 'scissors'){
-    return 'hand1 wins';
+    return "Hand one wins!";
   }
   else if(hand1 === 'rock' && hand2 === 'rock') {
-    return'its a tie';
+    return "It's a tie!";
   }
   else if(hand1 === 'paper' && hand2 === 'rock'){
     return 'hand1 wins';
   }
   else if(hand1 === 'paper' && hand2 === 'scissors'){
-    return'hand2 wins';
+    return "Hand two wins!";
   }
   else if(hand1 === 'paper' && hand2 === 'paper'){
-    return 'its a tie';
+    return "It's a tie!";
   }
-  else if(hand1 === scissors && hand2 === paper){
+  else if(hand1 === 'scissors' && hand2 === 'paper'){
     return 'hand1 wins';
   }
-  else if(hand1 === scissors && hand2 === rock ){
+  else if(hand1 === 'scissors' && hand2 === 'rock'){
     return 'hand2 wins';
   }
-  else{
-    return 'its a tie';
+  else if(hand1 === 'scissors' && hand2 === 'scissors'){ 
+    return "It's a tie!";
   }
-
 }
 
```

Comment:
Hi Jessica, how might we condense some work and make one else if statement for every tie?